### ISSUE TITLE
Specimen.analyte_concentration_method name

### ISSUE THREAD

**Issue by hsolbrig** [cancerDHC/data-model-harmonization#18]
This name is confusing -- this should be the method to _measure_ the analyte concentration, not the means of _concentrating_ the analyte

**Comment by bfurner** [comment]
@mbrush can you have a think about this?

**Comment by mbrush** [comment]
I agree with Harold's assessment. I am trying to keep fields names as short as possible but still clear/unambiguous. This can be hard when the model is so denormalized, but I digress.  

For this specific issue, lets consider the following alternatives to "analyte_concentration_method" (keeping in mind that this field is paired with the "analyte_concentration" field that holds the measured concentration): 
- "analyte_concentration_measurement_method"
- "analyte_concentration_determination_method"
- "analyte_concentration_determined_by"

These are all a mouthful, but more clearly describe what the field captures.
